### PR TITLE
Write AI data into InfluxDB

### PR DESCRIPTION
**Pull request by SuperZooper3**
Made a basic system to write data that the AI has given about people crossing and total people into an InfluxDB database

### CODE REVIEW

File: frontend/crowdeye/influx.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from influxdb import InfluxDBClient
+import datetime
+import pytz
+
+
+def writeEntry(nodeID, crossed_left, crossed_right, total_crossed, total_people):
+    """
+    writeEntry is a function to write data read the from the nodes into an InfluxDB database \n
```
Comment (tommcn) [inline frontend/crowdeye/influx.py:8]: No `\n` needed

File: frontend/crowdeye/management/commands/startserver.py [hunk 2]
```diff
@@ -33,11 +35,12 @@ def get_cams():
 
 
             print('t', DATA_LIST)
+            writeEntry(DATA_LIST[0]["node_id"], DATA_LIST[0]["crossed_left"], DATA_LIST[0]["crossed_right"], DATA_LIST[0]["total_crossed"], DATA_LIST[0]["total_people"])
```
Comment (tommcn) [inline frontend/crowdeye/management/commands/startserver.py:38]: Would their be a better way to do this

File: frontend/crowdeye/influx.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    The params are, in order: nodeID, crossed_left, crossed_right, total_crossed, total_people
+    """
+    #Get the current time in the RFC3339 format for InfluxDB
+    t = datetime.datetime.utcnow()
+    t = t.isoformat("T") + "Z" #Z is for Zulu timezone, idk why I need it but stackoverflow says to
```
Comment (tommcn) [inline frontend/crowdeye/influx.py:13]: Space after `#`

**Review by tommcn — CHANGES_REQUESTED** [review]

File: frontend/crowdeye/management/commands/startserver.py (hunk 2 above)
Comment (SuperZooper3) [reply frontend/crowdeye/management/commands/startserver.py]: Also need to make it possible to send the entire data list as there may be multiple nodes

**Comment by SuperZooper3** [comment]
It should be tested with multiple nodes to be sure that there are no problems. I can't test caus I only have 1 camera. @tommcn would be great if you could

**Review by tommcn — APPROVED** [review]

**Review by tommcn — APPROVED** [review]

**Pull request merged**